src/docfx/lib/markdown/XrefExtension.cs

```diff
@@ -38,23 +40,22 @@ internal static class XrefExtension
                     }
                     else if (node is HtmlBlock block)
                     {
+                        // inside html block
+                        // <p> <xref href="" uid=""/> </p>
                         block.Lines = new StringLineGroup(ResolveXref(block.Lines.ToString(), block));
                     }
                     else if (node is HtmlInline inline)
                     {
+                        // inside html inline
+                        // text <xref href="" uid=""/> text
                         inline.Tag = ResolveXref(inline.Tag, inline);
                     }
                     return node;
                 });
             });
 
             string ResolveXref(string html, MarkdownObject block)
-            {
-                return HtmlUtility.TransformXref(
-                    html,
-                    (href, isShorthand, columnOffset) => resolveXref(
-                        new SourceInfo<string>(href, block.ToSourceInfo(columnOffset: columnOffset)), isShorthand));
-            }
+                => HtmlUtility.TransformXref(html, block, resolveXref);
```

Comment:
Don't remove `columnOffset`, it is used to generate two errors if the same xref appears on the same line